home/import_assessments.py

```diff
@@ -185,6 +214,21 @@ def create_shadow_assessment_from_row(
         )
         assessment.questions.append(question)
 
+    def validate_headers(
```

Comment:
Do we have to pass in MANDATORY_HEADERS here? validate_headers is in the same class where it is called and can also access the variable directly.